docs/conf.py

```diff
@@ -112,64 +113,51 @@
 
 
 # -- Options for LaTeX output ---------------------------------------------
-
-latex_elements = {
-    # The paper size ('letterpaper' or 'a4paper').
-    #
-    # 'papersize': 'letterpaper',
-
-    # The font size ('10pt', '11pt' or '12pt').
-    #
-    # 'pointsize': '10pt',
-
-    # Additional stuff for the LaTeX preamble.
-    #
-    # 'preamble': '',
-
-    # Latex figure (float) alignment
-    #
-    # 'figure_align': 'htbp',
-}
+#
+# These options may be included in the dictionary below:
+#   The paper size ('letterpaper' or 'a4paper').
+#   'papersize': 'letterpaper',
+#
+#   The font size ('10pt', '11pt' or '12pt').
+#   'pointsize': '10pt',
+#
+#   Additional stuff for the LaTeX preamble.
+#   'preamble': '',
+#
+#   Latex figure (float) alignment
+#   'figure_align': 'htbp',
+latex_elements = {}
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
-latex_documents = [
-    (master_doc, 'pysatModels.tex', u'pysatModels Documentation',
-     u'Angeline G Burrell, Russel Stoneback, and Jeffery Klenzing', 'manual'),
-]
+latex_documents = [(master_doc, 'pysatModels.tex', u'pysatModels Documentation',
```

Comment:
Should we use `title = 'pysatModels Documentation'` (similar to authors) to aid in maintenance?